README.md

```diff
@@ -1 +1,142 @@
-# HelloMicroverse
\ No newline at end of file
+📗 Table of Contents
+
+    📖 About the Project
+        🛠 Built With
+            Tech Stack
+            Key Features
+        🚀 Live Demo
+    💻 Getting Started
```

Comment:
- [ ] See this [readme template](https://github.com/microverseinc/readme-template/blob/master/README.md) how it is showing and compare it to yours, Your readme file content should not present like that, like the codes you see, please make it sure you are using the correct copied text.

![5](https://user-images.githubusercontent.com/77843050/212100187-eb77fe11-adee-4ee6-bf5c-ceb6069b0e81.gif)

Reply:
Hello @joskalenda

Can you be more specific about which parts should I change in the README file?

Thank you